rust/gitxetcore/src/git_integration/git_repo_plumbing.rs

```diff
@@ -45,6 +44,21 @@ pub fn repo_dir_from_repo(repo: &Arc<Repository>) -> PathBuf {
     .to_path_buf()
 }
 
+pub fn normalize_repo_path(path: &str) -> String {
```

Comment:
Can we add a comment to describe the intent of this function?

Reply:
Sure.